presto-docs/src/main/sphinx/develop/spi-overview.rst

```diff
@@ -30,61 +31,63 @@ single line listing the name of the plugin class:
 
     com.facebook.presto.example.ExamplePlugin
 
+For a built-in plugin that is included in the Presto source code,
```

Comment:
This Maven packaging is provided by [1] that we recommend every plugin uses, as performs additional checks such as validating SPI dependencies. I'll update this in a future PR.

1. https://github.com/prestodb/presto-maven-plugin